### PR TITLE
[LM-9897] Peer Testing Bugs

### PR DESCRIPTION
**Pull request by arnavr15**
## Description

**Bugs**
- LocalDB bugs

**What kind of change does this PR introduce?** (check at least one)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: ChatSX/components/AttachmentConversations/index.tsx [hunk 1]
```diff
@@ -679,6 +679,26 @@ export const ImageConversations = ({
     }
   };
 
+  let imageSource = null;
+
+  if (attachment0) {
```
Comment (ishaan1607) [inline ChatSX/components/AttachmentConversations/index.tsx:684]: rename `attachment0` to `firstAttachment`

File: ChatSX/components/HomeFeedItem/index.tsx [hunk 2]
```diff
@@ -140,12 +140,13 @@ const HomeFeedItem: React.FC<Props> = ({
...
     let videosCount = 0;
     let pdfCount = 0;
+
     for (let i = 0; i < attachments.length; i++) {
       if (attachments[i].type == 'image') {
         imageCount++;
       } else if (attachments[i].type == 'video') {
         videosCount++;
-      } else {
+      } else if (attachments[i].type == 'pdf') {
```
Comment (ishaan1607) [inline ChatSX/components/HomeFeedItem/index.tsx:149]: create enums for 'pdf', 'video' etc

File: ChatSX/components/InputBox/index.tsx [hunk 3]
```diff
@@ -695,6 +699,10 @@ const InputBox = ({
           type: UPDATE_CHAT_REQUEST_STATE,
           body: {chatRequestState: 1},
```
Comment (ishaan1607) [inline ChatSX/components/InputBox/index.tsx:700]: replace `1` to enums created

File: ChatSX/constants/Styles.ts [hunk 4]
```diff
@@ -12,6 +12,7 @@ export default {
   $FONT_SIZES: {
     XS: 10,
     SMALL: 12,
+    PREMEDIUM: 13,
```
Comment (ishaan1607) [inline ChatSX/constants/Styles.ts:15]: rename to regular

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

**Review by ishaan1607 — APPROVED** [review]

**Review by jaixchaudhary — APPROVED** [review]

**Pull request merged**